src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions/StreamExtensions.cs

```diff
@@ -130,12 +126,32 @@ public static string ReadToEnd(this Stream stream)
 		/// <remarks>The stream will be disposed when calling this method.</remarks>
 		public static string ReadToEnd(this Stream stream, Encoding encoding)
 		{
-			string value;
-			using (var reader = new StreamReader(stream, encoding))
-			{
-				value = reader.ReadToEnd();
-			}
-			return value;
+			using var reader = new StreamReader(stream, encoding);
+			return reader.ReadToEnd();
+		}
+
+		/// <summary>
+		/// Reads the text container into the specified stream.
+		/// </summary>
+		/// <param name="stream"></param>
+		/// <returns>The string using the default encoding.</returns>
+		/// <remarks>The stream will be disposed when calling this method.</remarks>
+		public static async Task<string> ReadToEndAsync(this Stream stream, CancellationToken ct)
+		{
+			using var reader = new StreamReader(stream);
+			return await reader.ReadToEndAsync();
+		}
+
+		/// <summary>
+		/// Reads the text container into the specified stream.
+		/// </summary>
+		/// <param name="stream"></param>
+		/// <returns>The string using the default encoding.</returns>
+		/// <remarks>The stream will be disposed when calling this method.</remarks>
+		public static async Task<string> ReadToEndAsync(this Stream stream, Encoding encoding, CancellationToken ct)
+		{
+			using var reader = new StreamReader(stream, encoding);
+			return await reader.ReadToEndAsync();
 		}
```

Comment:
Are these unused?

Reply:
I used them in my tests and I kept them as they are `internal` and actually the only ones that are working on WASM (where files cannot be read synchronously). I would consider them as "debug helpers".